subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ComponentMetadataRulesDependenciesIntegrationTest.groovy

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.integtests.resolve
+
+import org.gradle.integtests.fixtures.AbstractHttpDependencyResolutionTest
+import org.gradle.test.fixtures.HttpRepository
+import org.gradle.test.fixtures.Module
+import spock.lang.Unroll
+
+abstract class ComponentMetadataRulesDependenciesIntegrationTest extends AbstractHttpDependencyResolutionTest {
```

Comment:
This test should probably be using `ResolveTestFixture`

Reply:
Thanks for the pointer @bigdaz. I was not aware of that fixture. Fits perfectly.